frontend/components/MediaContent/MediaContent.tsx

```diff
@@ -33,8 +32,7 @@ type MediaDetails = {
 
 export default function MediaContent({ media, alt }: Props) {
   const [hasWindow, setHasWindow] = useState(false);
-
-  const altText2 = alt === "" ? media[0].value.img.alt : alt;
+  const altText = alt;
```

Comment:
@MariaBee3 ,
a minor detail, but is there a reason for storing the prop to a a const and use the const  in line 65?
Because you don't do anything with the const, my suggestion would be to remove this line and use the prop in line65